test/unit/specs/store/user.spec.js

```diff
@@ -219,17 +230,23 @@ describe(`Module: User`, () => {
       }
     }))
 
-    // we need to force resetting of the store modules to enable the new electron mock
+    // we need to reset the module to use the mocked electron dependency
     jest.resetModules()
-    let setup = require(`../../helpers/vuex-setup`).default
-    let instance = setup()
-    let test = instance.shallow()
-    store = test.store
-    node = test.node
+    let { actions, state } = require(`renderer/vuex/modules/user.js`).default({
```

Comment:
`userModule` instead ?

Reply:
It was important to reset the module in this case to import the new mocked dependencies (see line 233).